riptide-spring-boot-autoconfigure/README.md

```diff
@@ -289,7 +289,7 @@ riptide:
         jitter: 25 milliseconds
       circuit-breaker:
         enabled: true
-        failure-threshold: 3 out of 5
```

Comment:
As you only add failure-ratio, can you please restore this line as example configuration?

Reply:
Agreed, I've restored this.